joebvp/VPmeasure.py

```diff
@@ -551,10 +543,11 @@ def go(specfilename, parfilename):
     main.show()
     app.exec_()
 
-def batch_fit(spec,filelist,outparfile=None,outmodelfile=None,**kwargs):
+def batch_fit(spec, filelist, outparfile=None, outmodelfile=None, **kwargs):
     '''
     Takes a number of input files and fits the lines in them.  The fitting algorithm will
-    be run until convergence for each input file.
+    be run until convergence for each input file. The program will then ask whether to run the failed
+    runs in GUI mode.
```

Comment:
We should include outparfile and outmodelfile in the docstring.  I can do it as well...

Reply:
Not sure what these do. Will leave a placeholder for them.